person.rb

```diff
@@ -0,0 +1,28 @@
+class Person
+  attr_accessor :name, :age
+  attr_reader :id
+
+  @id_tem = 0
+
+  class << self
+    attr_accessor :id_tem
+  end
+
+  def initialize(age, name = 'unknown', parent_permission: true)
+    self.class.id_tem += 1
+    @id = self.class.id_tem
+    @age = age
+    @name = name
+    @parent_permission = parent_permission
+  end
+
+  def can_use_services?
+    of_age? || @parent_permission
+  end
+
+  private
+
+  def of_age?
+    @age > 18
+  end
```

Comment:
@Anand8317 , Just as the previous reviewer has suggested, it is a requirement that the off_age? method returns true if the user is 18 years and above (@age >= 18) but your implementation returns true only if a user is above 18 years (@age >18). this means it returns false if the user is 18 years which doesn't align with the project requirements😉, kindly do well to make the changes😉.

<img width="762" alt="image" src="https://github.com/Anand8317/library/assets/64643995/743027d4-5b7f-4f8f-9f76-be2c10f0bca4">

```suggestion
  def of_age?
    @age >= 18
  end
```

Reply:
oh my god. Totally embarrassed. I misread it. Thank you.

Reply:
You don't have to be embarrassed😅 we are all learning and we might encounter such little mistakes as we progress 😅 But let's make better mistakes as we grow together.

Success on your journey.

Happy coding👍.